backend/api/v2beta1/run.proto

```diff
@@ -271,45 +271,65 @@ message PipelineTaskDetail {
   // System-generated ID of a task.
   string task_id = 2;
 
+  // Name of the corresponding pod assigned by the orchestration engine.
+  // Also known as node_id.
+  string pod_name = 3;
+
   // User specified name of a task that is defined in
   // [Pipeline.spec][].
-  string display_name = 3;
+  string display_name = 4;
```

Comment:
Protobuf field number can't be changed. We'll have to add new field with the next available number.

https://stackoverflow.com/questions/65230623/safeness-of-changing-proto-field-number

Reply:
Thank you. Fixed.